.github/workflows/linux.yml

```diff
@@ -12,21 +12,21 @@ jobs:
     strategy:
       matrix:
         perl-version:
-          - '5.22'
+          - '5.26'
           - '5.32'
           - '5.34'
           - '5.36'
     container:
       image: perl:${{ matrix.perl-version }}
     steps:
       - uses: actions/checkout@v2
-      - name: perl -V
-        run: perl -V
-      - name: Install early dep
-        run: cpanm -n File::ShareDir::Install
+      - name: dev packages for testing
+        run: apt update && apt install -y libgd-dev graphviz libxml2-dev
+      - name: Install early + extra deps
+        run: cpanm -n File::ShareDir::Install DBD::SQLite CGI
```

Comment:
Would it be better to add DBD::SQLite CGI to the develop-requires? FSDI needs to be there so the `Makefile.PL` will run.